registry/client/repository_test.go

```diff
@@ -100,7 +100,7 @@ func addTestCatalog(route string, content []byte, link string, m *testutil.Reque
 func TestBlobDelete(t *testing.T) {
 	dgst, _ := newRandomBlob(1024)
 	var m testutil.RequestResponseMap
-	repo, _ := reference.WithName("test.example.com/repo1")
+	repo, _ := reference.CreateNamed("", "test.example.com/repo1")
```

Comment:
yeah, let me update these, I did a replacement without seeing this was being tested.